src/main/resources/lang/he_IL.json

```diff
@@ -219,7 +219,8 @@
     "showSkyblockItemId": "Show Skyblock Item ID on Hover",
     "showFeatureNamesOnHover": "Show Feature Names on Hover",
     "resetSalvagedEssencesAfterLeavingMenu": "Reset Salvaged Essence Counter After Closing Menu",
-    "changeDungeonMapZoomWithKeyboard": "Adjust zoom with +/- keys"
+    "changeDungeonMapZoomWithKeyboard": "Adjust zoom with +/- keys",
+    "showProfileTypeInChat": "הוסף סוג פרופיל לצאט"
```

Comment:
Please revert this change and translate the string on Crowdin after the PR is merged. https://crowdin.com/project/skyblockaddons

Reply:
Done